### ISSUE TITLE
BotSkills update does not correctly generate an updated dispatch file

### ISSUE THREAD

**Issue by pavolum** [microsoft/botframework-solutions#3480]
#### What project is affected?
BotSkills CLI V1.0.15

#### What language is this in?
reproed with a c# bot but I imagine the botSkills bug applies to both TS and C# VAs

#### What happens?
```
botskills update --botName <YOUR_BOT_NAME> --remoteManifest "https://<YOUR_SKILL_NAME>.azurewebsites.net/manifest/manifest-1.1.json" --luisFolder <YOUR_LUIS_FOLDER_PATH> --cs
```
This command is actually builds a file with below name, in the skill's Luis folder
`<YOUR_SKILL_NAME>(1).luis'
note the "(1)" included in the file name

but again down the line, the script seems to refer the older version with name
`<YOUR_SKILL_NAME>.luis'
without the "(1)" in the file name

**eventually not updating the dispatch**. In fact everytime you run the update it increments to a new file, but should have actually replaced the .luis file? 

**The root of the issue is that if you follow the below repro steps, an updated dispatch file is not created. This may have something to do with the file name incrementation as outlined above but that has not been verified**

#### What are the steps to reproduce this issue?

- Create a fresh VA and Skill and connect the skill to the VA
- Add a new intent to the skills .lu files (or through the portal) and run update_cognitive_models.ps1
- Run the ```botskills update``` command to update the VAs dispatch file so it contains the new intent added to the skill
- Note that the dispatch file is not updated and a new .luis file with a number suffix is instead created in the skills deployment folder
- My suspicion is that the creation of the new .luis file in the skill proj as opposed to the previous behavior that replaced the exisiting .luis file is leading to the dispatch issue

#### What were you expecting to happen?
Dispatch to be updated with the new skill intent and the existing .luis file in the skill to be updated as opposed to an updated copy with a new name being created.

**Comment by Batta32** [comment]
Thanks @pavolum, we will be reviewing this issue. As soon as we have any update we will back to this thread 😊.

**Comment by Batta32** [comment]
@pavolum, this issue was solved in PR #3483 and it will be present in the next release.

I will **close** this issue, feel free to re open it.
